addons/website_livechat/static/tests/embed/chat_request.test.js

```diff
@@ -1,36 +0,0 @@
-import { describe, test } from "@odoo/hoot";
-import { loadDefaultEmbedConfig } from "@im_livechat/../tests/livechat_test_helpers";
-import { contains, start, startServer } from "@mail/../tests/mail_test_helpers";
-import { Command, patchWithCleanup, serverState } from "@web/../tests/web_test_helpers";
-import { session } from "@web/session";
-import { defineWebsiteLivechatModels } from "../website_livechat_test_helpers";
-
-describe.current.tags("desktop");
-defineWebsiteLivechatModels();
-
-test("chat request opens chat window", async () => {
```

Comment:
Note: Not robust, simulating server return and redundant with the `test_chat_request_flow_with_rating_ui` tour which tests the same flow but with real server data. Basically, we only want to see that chat requests from an operator opens a chat window on the visitor side and that's covered by the tour I mentioned.